package.json

```diff
@@ -18,10 +19,12 @@
     "react-scripts": "3.4.1"
   },
   "scripts": {
-    "start": "PORT=8080 react-scripts start",
+    "start": "concurrently \"yarn runReact\" \"yarn runServer\"",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "runReact": "BROWSER=none PORT=8081 react-scripts start",
+    "runServer": "PORT=8082 node server/index.js"
```

Comment:
these let you run both your servers via a yarn command (e.g. `yarn runReact`). The `yarn start` command will now just run both servers. You'll probably want to also install a node watcher so you don't have to restart node after every change.